akd/src/tree_node.rs

```diff
@@ -810,6 +546,30 @@ impl TreeNode {
 
 /////// Helpers //////
 
+pub(crate) fn get_longest_common_prefix(nodes: &[akd_core::Node]) -> NodeLabel {
```

Comment:
nit, we should be able to write this as 

```rust
nodes: &[crate::Node]
```

without directly referencing `akd_core` due to the re-exports of types. This will be clearer for devs of this specific crate.